bpf/node_config.h

```diff
@@ -82,3 +103,14 @@ DEFINE_IPV6(HOST_IP, 0xbe, 0xef, 0x0, 0x0, 0x0, 0x0, 0x0, 0x1, 0x0, 0x0, 0xa, 0x
 #define MONITOR_AGGREGATION 5
 #define MTU 1500
 #define ENABLE_IPSEC
+
+#ifdef ENABLE_MASQUERADE
+#define CT_MAP_TCP6 test_cilium_ct_tcp6_65535
```

Comment:
`ENABLE_MASQUERADE` should never be set for endpoints. That said, it might have worked just like this because it was using global table in both cases. Will double check.